src/System.IO.MemoryMappedFiles/src/Microsoft/Win32/SafeMemoryMappedFileHandle.Unix.cs

```diff
@@ -31,18 +38,21 @@ public sealed partial class SafeMemoryMappedFileHandle : SafeHandle
         internal readonly long _capacity;
 
         /// <summary>Initializes the memory-mapped file handle.</summary>
-        /// <param name="fileHandle">The underlying file handle; this may be null in the case of a page-file backed memory-mapped file.</param>
+        /// <param name="mapName">The name of the map; may be null.</param>
+        /// <param name="fileHandle">The underlying file handle; may be null.</param>
         /// <param name="inheritability">The inheritability of the memory-mapped file.</param>
         /// <param name="access">The access for the memory-mapped file.</param>
         /// <param name="options">The options for the memory-mapped file.</param>
         /// <param name="capacity">The capacity of the memory-mapped file.</param>
         internal SafeMemoryMappedFileHandle(
+            string mapName,
             SafeFileHandle fileHandle, HandleInheritability inheritability,
```

Comment:
Thanks. That style is used in files elsewhere, and since I've not actually added those lines here, I'm going to leave it as is.  We can do a separate formatting cleanup if we decide we want to fix all those.